src/SupportedVideoProviders.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Drupal\itk_video;
+
+/**
+ * Supported video providers.
+ */
+enum SupportedVideoProviders: string {
```

Comment:
Enum names are usually singular:


```suggestion
enum SupportedVideoProvider: string {
```

Reply:
Changed